src/NzbDrone.Core/Indexers/BroadcastheNet/BroadcastheNetParser.cs

```diff
@@ -50,7 +50,7 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                 throw new IndexerException(indexerResponse, "Indexer API call returned an error [{0}]", jsonResponse.Error);
             }
 
-            if (jsonResponse.Result.Results == 0)
+            if (jsonResponse.Result.Results == 0 || jsonResponse.Result?.Torrents?.Values == null)
```

Comment:
`jsonResponse.Result` will always have a value if it hits this condition since `jsonResponse.Result.Results` was called first